tutorial/westeros/westeros_historical_new_capacity.ipynb

```diff
@@ -0,0 +1,1082 @@
+    "The \"historical time-periods\" are all the years which have been defined prior to the `firstmodelyear`.\n",
+    "These can be used for two purposes.\n",
+    "\n",
+    "1. For **calibration of the model**: The `firstmodelyear` is used to specify as of which time-period the \"optimization\" should be carried out.\n",
+    "There will be some past dependencies though, such as technology investments, their activity or emissions amongst others.\n",
+    "In order to account for these dependencies, we can use the parameters \"historical_new_capacity\", \"historical_capacity\" and \"historical_emissions\" respectively.\n",
+    "These will have implications on dynamic constraints on capacities or activity.\n",
+    "In the case of emissions, `historical_emissions` can be accounted for in cumulative emission bounds.\n",
+    "2. The second application is when running **complex scenario setups**.\n",
+    "If for example the aim is to assess climate mitigation efforts based on achieving the NDCs in 2030, then two scenarios can be run.\n",
+    "The first scenario implements \"only\" the NDCs related policies up to 2030.\n",
+    "After 2030, there is no continuation of policies assumed, hence the model will return to a \"baseline\" trajectory (see *\\\"NDC\\\"* in the panel a. of the figure below).\n",
+    "This scenario will then serve as a basis for further analysis.\n",
+    "In order to avoid an \"overachievement\" of the NDCs, the time-periods until 2030 should be *fixed*.\n",
+    "The historical time-periods (see *\\\"NDC-mitigation\\\"* in panel b. in the figure below) are therefore shifted.\n",
+    "This can be done in the process of cloning the NDC scenario for use with mitigation runs.\n",
+    "In the process of [cloning](https://docs.messageix.org/en/stable/api.html?highlight=clone#message_ix.Scenario.clone) a scenario, the argument `shift_first_model_year=<year>` with the corresponding time-period can be passed.\n",
+    "In the process of cloning the scenario, for the example below, all time-periods prior to 2035 will be set to \"historical\" time-periods.\n",
+    "Results (\"VAR\") for those year will be added to the corresponding \"historical\" parameters.\n",
+    "    \n",
+    "<img src='_static/historical_new_capacity_use_in_scenarios.png' width='600'>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Historical time-periods in our scenario"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "years = scenario.set('year').tolist()\n",
+    "print(f\"The model years include: {years}\")\n",
+    "print(f\"the `firstmodelyear` is {scenario.firstmodelyear}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define generic Functions"
```

Comment:
a minor issue related to capitalizing the words in titles: "Defining generic functions"